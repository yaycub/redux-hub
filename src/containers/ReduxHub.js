import React from 'react';
import SearchForm from '../components/reduxHub/searchForm';
import UserInfo from '../components/reduxHub/userInfo';

const ReduxHub = () => {

  return (
    <>
      <SearchForm />
      <UserInfo />
    </>
  );
};

export default ReduxHub;
