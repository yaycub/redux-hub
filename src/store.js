import { createStore } from 'redux';
import thunk from 'redux-thunk';



