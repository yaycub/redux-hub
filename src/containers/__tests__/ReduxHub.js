import React from 'react';
import { mount } from 'enzyme';
import { Provider } from 'react-redux';
import store from '../../store';
import ReduxHub from '../ReduxHub';

describe('Redux Hub container', () => {
  it('should render Redux Hub', () => {
    const wrapper = mount(
      <Provider store={store} >
        <ReduxHub />
      </Provider>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
