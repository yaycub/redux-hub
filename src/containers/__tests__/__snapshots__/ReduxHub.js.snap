// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Redux Hub container should render Redux Hub 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <ReduxHub>
    <SearchForm>
      <form
        onSubmit={[Function]}
      >
        <input
          onChange={[Function]}
          type="text"
          value="Please Enter a Username"
        />
        <button>
          Search User
        </button>
      </form>
    </SearchForm>
    <UserInfo>
      <p>
        Loading...
      </p>
    </UserInfo>
  </ReduxHub>
</Provider>
`;
